tests/scipy_stats_test.py

```diff
@@ -564,6 +564,31 @@ def testIssue972(self):
       lsp_stats.norm.cdf(np.full((4,), np.inf, np.float32)),
       check_dtypes=False)
 
+  @parameterized.named_parameters(jtu.cases_from_list(
+      {"testcase_name": jtu.format_test_name_suffix("", [shape], [dtype]),
+       "dtype": dtype,
+       "shape": shape}
+      for shape in [(2,), (3,), (4,), (5,), (6,), (7,), (1,2), (1, 3), (1,4), (1,5)]
+      for dtype in itertools.combinations_with_replacement(jtu.dtypes.floating, 2)))
+  def testMultinomialLogPmf(self, shape, dtype):
+    rng = jtu.rand_positive(self.rng())
+    scipy_fun = osp_stats.multinomial.logpmf
+    lax_fun = lsp_stats.multinomial.logpmf
+
+    def args_maker():
+      x = np.rint(rng(shape, dtype[0])).astype(dtype[0])
```

Comment:
If inputs are expected to be integers, would it make more sense to generate integers directly?

Reply:
np.rint method does the same rounding as np.around or np.fix. We do not change to an integer because of the granularity of delta. Thank you!